front-end/src/components/ContactsTable.tsx

```diff
@@ -43,6 +74,14 @@ export default function ContactsTable({
                       <td className="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
                         {contact.fullName}
                       </td>
+                      <td className="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
+                        <button
+                          onClick={() => sendMessage(contact.id)}
+                          className={"rounded gap-x-3 border border-transparent py-2 px-3 text-sm font-semibold bg-gray-200 hover:bg-gray-300 text-gray-800"}
+                        >
+                          Send
+                        </button>
```

Comment:
Could be nice to add loading state (that also disabled the button) and a success message